lib/custom_widgets/fifth_custom_product.dart

```diff
@@ -0,0 +1,164 @@
+import 'package:flower_app/utils/constant_colors.dart';
+import 'package:flutter/material.dart';
+
+class FifthCustomProduct extends StatefulWidget {
+  final Color color;
+  final String title;
+  final String image;
+  final String image2;
```

Comment:
Wrong Naming Convention, Never use this thing again [image1, image2] never use numbering for naming convention, you can give them names according to their roles like backgroundImage, ForegraoundImage, CentereImage like this or you can think better according to UI